pkg/storaged/block/format-dialog.jsx

```diff
@@ -423,6 +429,8 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
             else if (trigger == "type") {
                 if (dlg.get_value("type") == "empty") {
                     dlg.update_actions({ Variants: action_variants_for_empty });
+                } else if (dlg.get_value("type") == "swap") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240118-122938-e6fc1c3f-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)